stdlib/public/Platform/winsdk.modulemap

```diff
@@ -244,6 +237,20 @@ module WinSDK [system] {
     link "shcore.lib"
   }
 
+  module System {
```

Comment:
I wonder if the deeply nested modules are really necessary.  I suppose that we can clean this up going forward.  It does make sense to try to at least get the headers split up better.

Reply:
> I wonder if the deeply nested modules are really necessary 

This is basically MSDN grouping, these headers are under a single section there. Is there a reason to specifically avoid nested modules?

> It does make sense to try to at least get the headers split up better.

`winioctl.h` doesn't include any major headers, what do you mean by splitting it?

Reply:
I just meant that extracting it from WInSock is good.